### ISSUE TITLE
include php5-curl 

### ISSUE THREAD

**Issue by predorock** [nurelm/prestashop_vagrant#14]
I have just intsalled the VM but i had to install php5-curl manually to allow prestashop to update modules.
So i think that this module should be inlcuded in the vagrant configuration file.
EDIT: also "sendmail" does't work a needs to be installed.

**Comment by shaabans** [comment]
Good idea.  Would you mind including php5-curl and, if you want to try it, sendmail in bootstrap.sh and submitting it via a pull request?
